integration/ct_hammer/main.go

```diff
@@ -0,0 +1,121 @@
+	getSTHConsistencyBias = flag.Int("get_sth_consistency", 2, "Bias for get-sth-consistency operations")
+	getProofByHashBias    = flag.Int("get_proof_by_hash", 2, "Bias for get-proof-by-hash operations")
+	getEntriesBias        = flag.Int("get_entries", 2, "Bias for get-entries operations")
+	getRootsBias          = flag.Int("get_roots", 1, "Bias for get-roots operations")
+	getEntryAndProofBias  = flag.Int("get_entry_and_proof", 0, "Bias for get-entry-and-proof operations")
+)
+
+func main() {
+	flag.Parse()
+	if *logConfigFlag == "" {
+		glog.Fatal("Test aborted as no log config provided")
+	}
+	if *seed == -1 {
+		*seed = time.Now().UTC().UnixNano() & 0xFFFFFFFF
+	}
+	rand.Seed(*seed)
+
+	cfg, err := ctfe.LogConfigFromFile(*logConfigFlag)
+	if err != nil {
+		glog.Fatalf("Failed to read log config: %v", err)
+	}
+
+	// Retrieve the test data.
+	caChain, err := integration.GetChain(*testDir, "int-ca.cert")
+	if err != nil {
+		glog.Fatalf("failed to load certificate: %v", err)
+	}
+	leafChain, err := integration.GetChain(*testDir, "leaf01.chain")
+	if err != nil {
+		glog.Fatalf("failed to load certificate: %v", err)
+	}
+	signer, err := integration.MakeSigner(*testDir)
+	if err != nil {
+		glog.Fatalf("failed to retrieve signer for re-signing: %v", err)
+	}
+	leafCert, err := x509.ParseCertificate(leafChain[0].Data)
+	if err != nil {
+		glog.Fatalf("failed to parse leaf certificate to build precert from: %v", err)
+	}
+	caCert, err := x509.ParseCertificate(caChain[0].Data)
+	if err != nil {
+		glog.Fatalf("failed to parse issuer for precert: %v", err)
+	}
+	bias := integration.HammerBias{Bias: map[ctfe.EntrypointName]int{
+		ctfe.AddChainName:          *addChainBias,
+		ctfe.AddPreChainName:       *addPreChainBias,
+		ctfe.GetSTHName:            *getSTHBias,
+		ctfe.GetSTHConsistencyName: *getSTHConsistencyBias,
+		ctfe.GetProofByHashName:    *getProofByHashBias,
+		ctfe.GetEntriesName:        *getEntriesBias,
+		ctfe.GetRootsName:          *getRootsBias,
+		ctfe.GetEntryAndProofName:  *getEntryAndProofBias,
+	}}
+
+	fmt.Print("\n\nStop ")
+	for i := 0; i < 8; i++ {
+		time.Sleep(100 * time.Millisecond)
+		fmt.Print(".")
+	}
+	fmt.Print(" Hammer Time\n\n")
+
+	type result struct {
+		prefix string
+		err    error
+	}
+	results := make(chan result, len(cfg))
+	var wg sync.WaitGroup
+	for _, c := range cfg {
+		wg.Add(1)
+		cfg := integration.HammerConfig{
+			LogCfg:     c,
+			MMD:        *mmdFlag,
+			LeafChain:  leafChain,
+			LeafCert:   leafCert,
+			CACert:     caCert,
+			Signer:     signer,
+			Servers:    *httpServersFlag,
+			EPBias:     bias,
+			Operations: *operationsFlag,
+		}
+		go func(cfg integration.HammerConfig) {
+			defer wg.Done()
+			err := integration.HammerCTLog(cfg)
+			results <- result{prefix: cfg.LogCfg.Prefix, err: err}
+		}(cfg)
+	}
+	wg.Wait()
+	close(results)
+	for e := range results {
+		if e.err != nil {
+			glog.Errorf("%s: %v", e.prefix, e.err)
```

Comment:
Would it be useful to note there was at least 1 failzor and `os.Exit(1)` down below?